tests/BrowsershotTest.php

```diff
@@ -1005,4 +1005,16 @@ public function it_can_input_form_fields_and_post_to_ssl_and_get_the_body_html()
             ],
         ], $command);
     }
+
+    /** @test */
+    public function it_can_write_options_to_a_file()
+    {
+        $targetPath = __DIR__.'/temp/testScreenshot.png';
+
+        Browsershot::url('https://example.com')
+            ->writeOptionsToFile()
+            ->save($targetPath);
+
+        $this->assertFileExists($targetPath);
+    }
```

Comment:
I would also add a test to assert that the PDF is actually generated with this options file